internal/controllers/cryostat_controller_test.go

```diff
@@ -213,6 +211,33 @@ var _ = Describe("CryostatController", func() {
 					})
 				})
 			})
+		}
+		Context("successfully creates required resources", func() {
+			BeforeEach(func() {
+				t.objs = append(t.objs, t.NewCryostat())
+			})
+
+			expectSuccessful()
+		})
+		Context("with multiple namespaces", func() {
+			namespaces := []string{"test-one", "test-two"}
+			BeforeEach(func() {
+				for _, ns := range namespaces {
+					t.Namespace = ns
+					t.objs = append(t.objs, t.NewNamespace(), t.NewCryostat())
```

Comment:
Changing `t.Namespace` changes the namespace for any subsequent test objects that are created, e.g. the namespace `t.NewNamespace`, and the namespace of `t.NewCryostat`. This was done in https://github.com/cryostatio/cryostat-operator/pull/507 to avoid adding namespace parameters for every test object. We might be able to improve on the design, but this is a compromise for now.